docs/Windows.md

```diff
@@ -31,10 +31,10 @@ The installation instructions that follow will result in a directory tree that h
 
 1. Go to [https://compnerd.visualstudio.com/swift-build](https://compnerd.visualstudio.com/swift-build).
 2. Choose `Pipelines` > `Pipelines` from the left of the dashboard.
-3. Choose 'Runs'.
-4. Use the filter symbol to search for "VS2017".
-5. In the results list, scroll down until you see the first successful build (with a green OK symbol) that does not have "facebook" in the description and click on it.
-6. Under "Artifacts", click on the "... published" link.
+3. Use the filter symbol to search for "VS2019".
+4. Click on the appropriate pipeline (e.g. `VS2019`).
+5. The list of the runs will be displayed, scroll down until you see the first successful build (with a green OK symbol) and click on it.
+6. Click the link under "Artifacts:".
 7. Download windows-toolchain-amd64.msi, windows-sdk.msi, and windows-runtime-amd64.msi by clicking on the appropriate down-arrows on the right. Be sure to really download these files from the same build (i.e. do not switch the build for the next download, and be careful when updating). These files will be downloaded as zip files. Unless they are not automatically unzipped during the download process, unzip them to obtain \*.msi files in the extracted directories.
```

Comment:
I think you’re in a better position than I to judge this: should this just be changed to use the swift-build utility to download instead?

At the very least we should clear up the fact that a failure isn’t always a failure - the VS2019 builds Android too, and that a couple of tests are flakey (like in libdispatch and foundation).  Let’s clear it up to indicate that those are safe to use.

Reply:
I will have a look at the swift-build utility, maybe add a 2nd possibility in the description, we should first make the current description work.

Reply:
Okay, doing that as a follow up is fine.